libfreerdp/primitives/sse/prim_templates.h

```diff
@@ -86,21 +83,21 @@
 		{                                                                             \
```

Comment:
I think we can ditch this check (and half of the code below) after the lddq/load changes